### ISSUE TITLE
Не восстанавливается Backup

### ISSUE THREAD

**Issue by 53RG1005** [serghey-rodin/vesta#2205]
Vesta 1 r.5

Нажимаю восстановить backup, появляется попап где я нажимаю ок и ничего не происходит (на первый взгляд)

Но в консоли вижу ответ сервера
{ok: null, error: "Задание уже выполняется, пожалуйста дождитесь окончания."}
error: "Задание уже выполняется, пожалуйста дождитесь окончания."
ok: null

Можно сделать, что бы это сообщение показывалось, т.к. не очевидно что это происходит, кроме того, подтверждения как такового нет, это формальный попап с кнопкой ОК, задание и без клика ОК отправляется в обработку

**Comment by anton-reutov** [comment]
Данные из копии восстанавливаются?

**Comment by 53RG1005** [comment]
Да, но без консоли этого непонятно, нужно добавить отображение статуса

{ok: null,…}
error: "Задание успешно добавлено в очередь. После завершения вы получите полный отчет по почте."
ok: null

**Comment by andykimpe1** [comment]
On GitHub, we only speak English.

English is the only language allowed.

This is part of the terms you have accepted.

By signing up here.

**Comment by Neustradamus** [comment]
Dear all,

@Dennis-SEG has done a very good job here:
- https://github.com/outroll/vesta/pull/2322

What do you think?
